spec/section/joins.md

```diff
@@ -1,116 +1,38 @@
-
-- a <dfn>parent map</dfn> (`rml:parentMap`), whose value is an <a data-cite="RML-Core#dfn-expression-map">expression map</a> (`rml:ExpressionMap`),
-  which, as the join condition's parent map, MUST include references that exist in the logical view specified by the parent logical view property, or it should have a constant value.
-
-The [=join condition=] returns true when values produced by the child map and the parent map during the iteration are equal.
-<aside class="note">
-If no data type is specified in the field, string values are compared.
-Data types are not taken into account. 
-`1.0` will not match with `1.00`. 
-To secure this match a transformation with <a href="https://kg-construct.github.io/rml-fnml/ontology/documentation/index-en.html">RML-FNML:Functions</a> needs to be configured. 
-TODO describe what happens if the data types are deduced from the source (refer to the description that will be added to rml:core? 
-</aside>
-
-<aside class="note">
-This definition is in line with the definition in RML CORE, with one small difference: it refers directly to a parent logicial source, and not to the logical source of the parent triples map.
-</aside>
-
-| Property                    | Domain               | Range                     |
-| --------------------------- | -------------------- | ------------------------- |
-| `rml:childMap`              | `rml:JoinCondition`  | `rml:ExpressionMap`       |
-| `rml:parentMap`             | `rml:JoinCondition`  | `rml:ExpressionMap`       |
-
-#### Shortcuts
-
-If the value of the [=child map=] property (`rml:childMap`) is a [reference-valued Expression Map](https://kg-construct.github.io/rml-core/spec/docs/#reference-rml-reference),
-then the `rml:child` shortcut could be used.
-
-Similarly, if value of the [=parent map=] (`rml:parentMap`) is a [reference-valued Expression Map](https://kg-construct.github.io/rml-core/spec/docs/#reference-rml-reference),
-then the `rml:parent` shortcut could be used.
-
-| Property                    | Domain               | Range                     |
-| --------------------------- | -------------------- | ------------------------- |
-| `rml:child`                 | `rml:JoinCondition`  | `Literal`                 |
-| `rml:parent`                | `rml:JoinCondition`  | `Literal`                 |
-
-<aside class="issue">
-Els: or can we just refer to rml core and not specify any definitions here?
-</aside>
-<aside class="issue">
-Els: can we also optionally declare a join function here, to allow not only equijoins (default) but also other joins
-</aside>
-
 ### Join types {#dfn-join-type}
 
-A [=logical view=] (`rml:LogicalView`) MAY have one or more with join properties, specifying the join type, i.e. a [=left join=] and a [=inner join=].
+A [=logical view=] (`rml:LogicalView`) MAY have one or more join properties, whose value is a [=logical view join=]. 
```

Comment:
This is also already stated in section 4. Logical Views. Ideally, we should only state it once, either here or in section 4.
If we need to reference "join properties" here, we should probably define the term.

Reply:
See my new commit: https://github.com/kg-construct/rml-lv/pull/36/commits/63046b29f38144835c7e549f290f9cb8df3c3747.